pkg/server/server.go

```diff
@@ -24,6 +24,9 @@ func ListenAndServe(ctx context.Context, cfg *rest.Config, clients *k8s.Clients,
 	}
 
 	router := mux.NewRouter()
+	authHandler := supportconfig.NewAuthHandler(clients.SAR, clients.TR)
+	// handle auth before allowing support config to be created
```

Comment:
(nit)

```suggestion
	// handle auth before allowing supportconfig to be created
```

A supportconfig is a suse thing, it's one word

Reply:
Resolving due to change in goals with this PR.